test/compilable/casttuple.d

```diff
@@ -0,0 +1,32 @@
+alias tuple(T...) = T;
+
+void exactMatch()
+{
+    tuple!int tup_1;
+
+    // https://issues.dlang.org/show_bug.cgi?id=19954
```

Comment:
does this PR fix this issue? if so please amend the _commit tile_ to say "fix issue 19954:..."

Reply:
This doesn't fix it, will remove the line. :+1: